### PR TITLE
how to add fonts to your linux server

### PR DESCRIPTION
**Pull request by tudorbarascu**

### CODE REVIEW

File: source/docs/user_manual/working_with_ogc/ogc_server_support.rst [hunk 1]
```diff
@@ -679,6 +679,43 @@ your :file:`*.conf` file look like::
...
+In order to ensure this does not happen you just need to install the missing fonts on the target machine.
+Doing this on desktop systems is usually trivial (double clicking the fonts).
+
+For linux, if you don't have a desktop enviroment installed (or you prefer the command line) you need to:
+
+* On Debian based systems::
+
+   sudo su
+   mkdir /usr/local/share/fonts/truetype && cd /usr/local/share/fonts/truetype
+   mkdir myfonts && cd myfonts
```
Comment (yjacolin) [inline source/docs/user_manual/working_with_ogc/ogc_server_support.rst]: why two line to create the directory and cd inside?
``mkdir /usr/local/share/fonts/truetype/myfonts && cd /usr/local/share/fonts/truetype/myfonts```

File: source/docs/user_manual/working_with_ogc/ogc_server_support.rst [hunk 2]
```diff
@@ -679,6 +679,43 @@ your :file:`*.conf` file look like::
      AllowOverride None
    .....
 
+.. _qgis-add-fonts:
```
Comment (yjacolin) [inline source/docs/user_manual/working_with_ogc/ogc_server_support.rst]: we use _ instead of -
Do we need to use qgis as a prefix? I propose `add_fonts`

**Comment by yjacolin** [comment]
LGTM except my two comments.

**Comment by tudorbarascu** [comment]
@yjacolin fixed and rebased. How's it looking now?

File: source/docs/user_manual/working_with_ogc/ogc_server_support.rst [hunk 3]
```diff
@@ -679,6 +679,42 @@ your :file:`*.conf` file look like::
...
+------------------------------
+
+Keep in mind that you may use QGIS projects that point to fonts that
+may not exist by default on other machines. This means that if you share the project,
+it may look different on other machines (if the fonts don't exist on the target machine).
+
+In order to ensure this does not happen you just need to install the missing fonts on the target machine.
+Doing this on desktop systems is usually trivial (double clicking the fonts).
+
+For linux, if you don't have a desktop enviroment installed (or you prefer the command line) you need to:
```
Comment (DelazJ) [inline source/docs/user_manual/working_with_ogc/ogc_server_support.rst]: environment

**Comment by DelazJ** [comment]
LGTM though you have some long lines :)

**Pull request merged**